virtual_rainforest/main.py

```diff
@@ -109,18 +192,39 @@ def vr_run(
         "All models successfully configured, now attempting to initialise them."
     )
 
-    # This is just a step to pass flake8 checks (DELETE LATER)
-    print(models_cfd)
+    # Define the basic model time grain (set to 10 minutes for now)
+    update_interval = timedelta64(10, "m")
```

Comment:
Is this something you want to hardcode rather than reading it from the configuration?

Reply:
Yeah my worry was with users setting an update interval that wasn't small enough to capture the timing of models well, i.e. setting it to be 3 weeks when one model updates every three weeks while another updates monthly would generate undesirable behaviour.

Can definitely see the value in this being configurable though, I'll probably add it to the config schema with the default set to 10 minutes, and the description for the user warning them to set it significantly lower than the model time step